offline.appcache

```diff
@@ -0,0 +1,44 @@
+CACHE MANIFEST
+# 2016-01-14 v1.1.3
+
+# Main HTML file
+index.html
+
+# Translations
+cz/index.html
+da/index.html
+de/index.html
+es/index.html
+fr/index.html
+ko/index.html
+pl/index.html
+pt/index.html
+ru/index.html
+zh/index.html
+
+# CSS Files
+css/critical.css
```

Comment:
Also the critical CSS file in only used when JavaScript is disabled, else it’s inlined. So I can confirm we don’t need it. :)